spec/rubocop/cop/style/mixin_grouping_spec.rb

```diff
@@ -45,14 +45,17 @@
       let(:message) { 'Put `include` mixins in separate statements.' }
 
       context 'with several mixins in one call' do
-        it_behaves_like 'code with offense', <<-END.strip_indent
-          class Foo
-            include Bar, Qux
-          end
-        END
+        it_behaves_like 'code with offense',
+                        ['class Foo',
+                         '  include Bar, Qux',
```

Comment:
Interesting, I did not know that. Thanks for the link to the comment. I will fix this.